ui-frontend/packages/catalog-ui-search/src/main/webapp/component/property/property.js

```diff
@@ -108,6 +108,10 @@ module.exports = Backbone.Model.extend({
     currentValue.sort()
     switch (this.getCalculatedType()) {
       case 'location':
+        const currentShape = currentValue[0].mode
```

Comment:
Why is this part necessary?

Reply:
I wanted to restrict triggering a render to shape changes in the location dropdown only. Otherwise, it was triggering a render everytime the drawing was changed.